packages/observable/src/subscribable.ts

```diff
@@ -3,38 +3,40 @@ import {
     arrayRemoveItem, objectForEach, options
 } from '@tko/utils'
 
-import Subscription from './Subscription'
-import { SUBSCRIBABLE_SYM } from './subscribableSymbol'
+import { default as Subscription, LATEST_VALUE } from './Subscription'
 import { applyExtenders } from './extenders.js'
 import * as dependencyDetection from './dependencyDetection.js'
 export { isSubscribable } from './subscribableSymbol'
+import { SUBSCRIBABLE_SYM } from './subscribableSymbol'
 
 // Descendants may have a LATEST_VALUE, which if present
 // causes TC39 subscriptions to emit the latest value when
 // subscribed.
-export const LATEST_VALUE = Symbol('Knockout latest value')
 
-export function subscribable () {
+export function subscribable<T> (this: Subscribable<T>) : void {
   Object.setPrototypeOf(this, ko_subscribable_fn)
   ko_subscribable_fn.init(this)
 }
 
-export var defaultEvent = 'change'
+interface TC39Callback {
+}
+type Subscriber = (value: any) => void | TC39Callback
 
-var ko_subscribable_fn = {
+export const defaultEvent = 'change'
+
+const ko_subscribable_fn = {
   [SUBSCRIBABLE_SYM]: true,
   [Symbol.observable] () { return this },
 
-  init (instance) {
+  init<T> (instance : Subscribable<T>) : void {
     instance._subscriptions = { change: [] }
     instance._versionNumber = 1
   },
 
-  subscribe (callback, callbackTarget, event) {
+  subscribe<T> (this: Subscribable<T>, callback: Subscriber, callbackTarget?: Function, event = defaultEvent) : Subscription {
```

Comment:
Avoid `Function` types. In fact, it should probably be added to tslint; I believe there is a `ban-types` option.